SQL/0000-00-01-Permission.sql

```diff
@@ -50,69 +50,66 @@ CREATE TABLE `user_perm_rel` (
-    (54,'publication_view', 'Publication - Access to module', 2),
-    (55,'publication_propose', 'Publication - Propose a project', 2),
-    (56,'publication_approve', 'Publication - Approve or reject proposed publication projects', 2),
-    (57, 'candidate_dob_edit', 'Edit dates of birth', 2),
-    (58,'electrophysiology_browser_view_allsites', 'View all-sites Electrophysiology Browser pages', 2),
-    (59,'electrophysiology_browser_view_site', 'View own site Electrophysiology Browser pages', 2),
-    (60,'battery_manager_view','Battery Manager: View Test Entries',2),
-    (61,'battery_manager_edit','Battery Manager: Edit Test Entries',2),
-    (62,'module_manager_view', 'Module Manager: access the module', 2),
-    (63,'module_manager_edit', 'Module Manager: edit installed modules', 2),
-    (64,'candidate_dod_edit', 'Edit dates of death', 2),
-    (65,'violated_scans_view_ownsite','Violated Scans: View Violated Scans from own site','2'),
+    (12,'examiner_view','Examiner Module Edit Own Site','2'),
+    (13,'examiner_multisite','Examiner Module Edit All Sites','2'),
+    (15,'timepoint_flag','Candidate Parameter Module Edit Exclusion Flags Own Site','2'),
+    (17,'conflict_resolver','Conflict Resolver Module Edit Own Site','2'),
+    (18,'data_dict_view','Data Dictionary Module View','2'),
+    (19,'violated_scans_view_allsites','MRI Violated Scan Module View All Sites','2'),
+    (22,'config','Configuration Settings Edit','2'),
+    (23,'imaging_browser_view_site','Imaging Browser Module View Own Site','2'),
+    (24,'imaging_browser_view_allsites','Imaging Browser Module View All Sites','2'),
+    (25,'dicom_archive_view_allsites','DICOM Module Archive Access','2'),
+    (28,'instrument_builder','Instrument Builder Module Edit Instruments','2'),
+    (29,'data_dict_edit','Data Dictionary Edit','2'),
+    (31,'candidate_parameter_view','Candidate Parameter Module View Own Site','2'),
+    (32,'candidate_parameter_edit','Candidate Parameter Module Edit Own Site','2'),
+    (33,'genomic_browser_view_site','Genomic Browser Module View Own Sites','2'),
+    (34,'genomic_browser_view_allsites','Genomic Browser Module View All Sites','2'),
+    (35,'document_repository_view','Document Repository Module View and Upload Own File','2'),
+    (36,'document_repository_delete','Document Repository Module Delete All Files','2'),
+    (37,'server_processes_manager','Server Process Module View','2'),
+    (38,'imaging_uploader','Imaging Uploader Module Upload Images for Own Site','2'),
+    (39,'acknowledgements_view','Acknowledgement Module View All Sites','2'),
+    (40,'acknowledgements_edit','Acknowledgement Module Edit All Sites','2'),
+    (41,'dataquery_view','Data Query Tool View All Sites','2'),
+    (42,'genomic_data_manager','Genomic Browser Module Edit Genomic Files at Own Site','2'),
+    (43,'media_write','Media Module Upload, Download, Edit All Sites','2'),
+    (44,'media_read','Media Module View File names own site','2'),
```

Comment:
these dont seem consistent with the rest. also not sure if you actually need write for being able to download